src/Models/User.php

```diff
@@ -90,6 +93,11 @@ public function getAvatarPathAttribute(): string
         return asset($this->attributes["avatar_path"]);
     }
 
+    public function codes(): HasOne
```

Comment:
```suggestion
    public function code(): HasOne
```
HasOne relationship should be in singular form.